charts/kube-ovn/templates/ovncni-ds.yaml

```diff
@@ -126,7 +126,9 @@ spec:
               - NET_BIND_SERVICE
               - NET_RAW
               - SYS_ADMIN
+              {{- if not .Values.DISABLE_MODULES_MANAGEMENT }}
               - SYS_MODULE
+              {{- end }}
```

Comment:
kube-ovn-cni will fail to start if we drop capability SYS_MODULE.

Reply:
Yeah you're right, this makes ovn totally incompatible with Talos Linux, because it is not allowed managing system modules by containers.

https://www.talos.dev/v1.7/learn-more/process-capabilities/